package.json

```diff
@@ -30,6 +30,7 @@
     "next-transpile-modules": "^7.3.0",
     "react": "17.0.1",
     "react-dom": "17.0.1",
+    "react-facebook-login": "^4.1.1",
```

Comment:
What is this module used for? If it is not required (and was just used for testing) please remove it.

Reply:
<img width="1440" alt="Screenshot 2021-11-07 at 6 05 31 PM" src="https://user-images.githubusercontent.com/40771676/140645204-14945f56-f828-4124-bb70-df26f3699ed6.png">
<img width="1440" alt="Screenshot 2021-11-07 at 6 05 41 PM" src="https://user-images.githubusercontent.com/40771676/140645213-6a5054a0-b4f7-4f82-953c-45437910f244.png">

I removed the "react-facebook-login" module, it's not required.